core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcDatabaseConditionalMutationIntegrationTest.java

```diff
@@ -32,25 +29,6 @@ protected int getThreadNum() {
     return super.getThreadNum();
   }
 
-  @Override
-  protected List<OperatorAndDataType> getOperatorAndDataTypeListForTest() {
-    if (rdbEngine == RdbEngine.ORACLE) {
-      List<OperatorAndDataType> ret = new ArrayList<>();
-      for (Operator operator : Operator.values()) {
-        for (DataType dataType : DataType.values()) {
-          // Oracle doesn't support conditions for BLOB type
-          if (dataType == DataType.BLOB) {
-            continue;
-          }
-          ret.add(new OperatorAndDataType(operator, dataType));
-        }
-      }
-      return ret;
-    }
-
-    return super.getOperatorAndDataTypeListForTest();
-  }
```

Comment:
After we changed the Blob type from `BLOB` to `RAW(2000)` in Oracle in the following PR, we support conditions for BLOB type.
https://github.com/scalar-labs/scalardb/pull/645/files#diff-d79cc6031633d67d741355267fef8120873fc11f76b10f911b7f38b2920b806dL86-R86

So we can remove this.